mixer/adapter/memquota/memquota.go

```diff
@@ -81,16 +81,14 @@ func limit(cfg *config.Params_Quota, instance *quota.Instance, l adapter.Logger)
 	for idx := range cfg.Overrides {
 		o := cfg.Overrides[idx]
 		if matchDimensions(o.Dimensions, instance.Dimensions) {
-			if l.VerbosityLevel(4) {
-				l.Infof("quota override: %v selected for %v", o, *instance)
-			}
+			l.Debugf("quota override: %v selected for %v", o, *instance)
```

Comment:
in some places, this PR replaces statements like `if l.VerbosityLevel(4)` with `if l.DebugEnabled()` and in other places, it removes the check on debug enabled. Is there any particular pattern for those choices? Is this just based on looking at the log statement and deciding that it isn't a perf burden to log?